docs/OPENSHIFT.md

```diff
@@ -0,0 +1,184 @@
+# LogDNA Agent on OpenShift
+
+The agent is supported for OpenShift 4.6.
+
+## Table of Contents
+
+* [Installing](#installing)
+  * [Installation Prerequisites](#installation-prerequisites)
+  * [Installation Steps](#installation-steps)
+* [Upgrading](#upgrading)
+  * [Upgrading the Configuration](#upgrading-the-configuration)
+    * [Upgrading from Configuration v1.x.x](#upgrading-from-configuration-v1xx)
+  * [Upgrading the Image](#upgrading-the-image)
+* [Uninstalling](#uninstalling)
+* [Run as Non-Root](#run-as-non-root)
+* [Collecting Node Journald Logs](#collecting-node-journald-logs)
+  * [Enabling Journald on the Node](#enabling-journald-on-the-node)
+  * [Enabling Journald Monitoring on the Agent](#enabling-journald-monitoring-on-the-agent)
+
+## Installing
+
+The agent can be installed in your cluster using a set of YAML files we provide. These files contain the minimum necessary OpenShift objects and settings to run the agent. Teams should review and modify these YAML files for the specific needs of their clusters.
+
+### Installation Prerequisites
+
+* LogDNA Account - Create an account with LogDNA by following our [quick start guide](https://docs.logdna.com/docs/logdna-quick-start-guide).
+* LogDNA Ingestion Key - You can find an ingestion key at the top of [your account's Add a Log Source page](https://app.logdna.com/pages/add-host).
+* OpenShift cluster running Kubernetes 1.9 or greater.
+* Local clone of this repository.
+
+### Installation Steps
+
+1. Navigate to the root directory of the cloned `logdna-agent-v2` repository.
+2. Run the following commands to create and configure a new project, secret, and service account:
+```console
+oc new-project logdna-agent
+oc create serviceaccount logdna-agent
+oc create secret generic logdna-agent-key --from-literal=logdna-agent-key=<YOUR LOGDNA INGESTION KEY>
+oc adm policy add-scc-to-user privileged system:serviceaccount:logdna-agent:logdna-agent
```

Comment:
Just testing if we can avoid this and get away with one of the less powerful SCCs. Watch this space

Reply:
Nope, doesn't look like it's possible, we may need a custom SCC or something. This is how fluentd does it, so at least we're not doing anything worse than they are

Reply:
It always need to run as privileged, non-root or not, it's what openshift requires if you want to access the host